contracts/remote-address-validator/RemoteAddressValidator.sol

```diff
@@ -27,7 +31,17 @@ contract RemoteAddressValidator is IRemoteAddressValidator, Upgradable {
     constructor(address _interchainTokenServiceAddress) {
         if (_interchainTokenServiceAddress == address(0)) revert ZeroAddress();
         interchainTokenServiceAddress = _interchainTokenServiceAddress;
-        interchainTokenServiceAddressHash = keccak256(bytes(_lowerCase(interchainTokenServiceAddress.toString())));
+        string memory interchainTokenServiceAddressString = interchainTokenServiceAddress.toString();
+        interchainTokenServiceAddressHash = keccak256(bytes(_lowerCase(interchainTokenServiceAddressString)));
```

Comment:
```suggestion
        interchainTokenServiceAddressHash = keccak256(bytes(nterchainTokenServiceAddressString));
```
since the string comes from `toString`, don't need to lowercase it again?